src/api/models/article.js

```diff
@@ -15,10 +15,10 @@ export default (sequelize, DataTypes) => {
         type: DataTypes.INTEGER
       },
       title: { type: DataTypes.STRING, allowNull: false },
-      description: { type: DataTypes.TEXT, allowNull: false },
-      body: { type: DataTypes.TEXT, allowNull: false },
+      description: { type: DataTypes.TEXT},
```

Comment:
@alainburindi why do you need to allow null here ?

Reply:
@alainburindi can you reply to this question here? Thanks